### ISSUE TITLE
Turntable footage

### ISSUE THREAD

**Issue by arstropica** [NVlabs/instant-ngp#1201]
Can colmap process turntable footage from a stationary camera?
Currently, I am getting zero image matches for 150 images of a 360 turntable sequence.

**Comment by toomy0toons** [comment]
yes it does and it should perform much better than handheld 360 camera video

try removing background or shooting object within non-transparent box
or in stable lighting

**Comment by avclubvids** [comment]
I got a successful turntable solve doing this:

1. Put some texture on the turntable surface so that COLMAP sees the "ground" rotating correctly with the subject. You might want to put a border around the turntable platter and basically turn it from a flat disc into a short cylinder. You can clip this surface out with the aabb box cropping later.
3. Shoot against as plain a background as possible. I used white but I would recommend black, as white ends up in the NeRF as a sphere of white clouds that you'll need to remove later.
4. Clean up anything in the photos that is not the color of the background (again, black is best here)
5. Process as normal, COLMAP should solve the cameras just fine.

**Comment by arstropica** [comment]
@avclubvids Thanks for the tips.  I will try those suggestions.
